frontend/public/components/graphs/query-browser.tsx

```diff
@@ -151,18 +152,24 @@ const QueryBrowser_: React.FC<QueryBrowserProps> = ({
 
   const samples = 300;
 
-  const urls = _.map(queries, query => getPrometheusURL({
-    endpoint: PrometheusEndpoint.QUERY_RANGE,
-    endTime,
-    query,
-    samples,
-    timeout: '5s',
-    timespan: span,
-  }));
-
   const safeFetch = useSafeFetch();
 
-  const tick = () => Promise.all(urls.map(safeFetch))
+  const safeFetchQuery = query => {
```

Comment:
general comment: we should ideally be adding types to parameters in our helpers

Reply:
Addressed by https://github.com/openshift/console/pull/1837